dDatabase/Tables/CDItemComponentTable.cpp

```diff
@@ -45,7 +45,7 @@ CDItemComponentTable::CDItemComponentTable(void) {
 		entry.offsetGroupID = tableData.getIntField(19, -1);
 		entry.buildTypes = tableData.getIntField(20, -1);
 		entry.reqPrecondition = tableData.getStringField(21, "");
-		entry.animationFlag = tableData.getIntField(22, -1);
+		entry.animationFlag = tableData.getIntField(22, 0);
```

Comment:
Default was changed to reflect how the animation flag works and that it's a uint and can't be negative

Reply:
However the field that takes this in serialization is a uint and 0 is the "None"/unused value for that field, so instead of having to do type casting later and accounting for it being -1, just make it be in a state when read that makes sens to how it is being used.

And AFAIK, the cdclient doesn't distinguish between uints and ints everything is simply a unit that is called in.

Reply:
That seems to be correct.